src/modules/gifmaker/GifRecorderOverlay/MainPage.xaml

```diff
@@ -19,6 +19,11 @@
             <AppBarButton x:Name="RecordPauseButton" Icon="Target" Label="Record" IsEnabled="False" Tag="record" Click="RecordPauseButton_Click"/>
             <AppBarButton x:Name="StopButton" Icon="Stop" Label="Stop" IsEnabled="False" Click="StopButton_Click"/>
         </CommandBar>
-
+        <Canvas x:Name="MainCanvas"
+                Background="Black"
```

Comment:
avoid literal colors for XAML.  Use {ThemeResource xxxxx} or {StaticResource xxxxx} values.